### DISCUSSION TITLE
Dev SQLite / Prod Postgresql

### DISCUSSION THREAD

**Discussion by mikeyakymenko** [prisma/prisma#16758] [Q&A] [answered]
Hi! I am a rookie with Prisma. I have a question and didn't find answer. 

I use `@prisma-utils/prismerge` to merge my files.

```
// base.prisma dev
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "sqlite"
  url = env("DB_URL")
}
```

and

```
// base.prisma prod
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url = env("DB_URL")
}
```

It's basically one line + migrations directory. How to handle it in most proper way?

**Comment by nurul3101** [comment, marked as answer]
Hey @mikeyakymenko 

If I understand correctly, do you want to configure your app so that the correct schema file should be used based on the `.env` file?
To elaborate, for production env file, the production schema file with PostgreSQL should be used, and for development env file, SQLite should be used?
Can you have a look at this guide on using [Multiple Env Files](https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files)?

Also, on a side note, `@prisma-utils/prismerge` is not officially maintained by Prisma team.

**Reply by mikeyakymenko** [reply]
@nurul3101 I used env files. Problem that DB type should be the same dev and prod. But I used different types, dev sqlite, prod postgres. And how manage this, cause Prisma says that "provider" couldn't be taken from env file.

**Reply by nurul3101** [reply]
I got it now what you are trying to achieve. Actually, this is considered as an Anti-pattern due to which Prisma doesn't support this - Please see this related [Issue](https://github.com/prisma/docs/issues/1233).

**Reply by mikeyakymenko** [reply]
I know that is anti-pattern, but I really need this atm :) tnx man!

**Reply by nurul3101** [reply]
Would overriding the datasource URL at runtime work for you: [Reference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#programmatically-override-a-datasource-url)?

**Reply by mikeyakymenko** [reply]
It's solve a dev environment problem, but doesn't solve the migrations. I guess it's no way to implement what I want. But I appreciate your help anyway. Thank you!

**Reply by anitacung** [reply]
👋 Hi @mikeyakymenko 

I hope you’re having a great start to 2023! ✨My name is Anita and I’m a Community Engagement Manager at Prisma. 

Thank you for creating this GitHub discussion, I recommend opening a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) if you believe we should include this in our roadmap. I also suggest commenting on the [docs issue ](https://github.com/prisma/docs/issues/1233)Nurul provided if you want to see more documentation about this issue.

In the meantime, feel free to join our [community Slack channel](https://slack.prisma.io/). This is a great way to network with other users and ask your Prisma ORM questions 😁

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
